leshan-server-core/src/main/java/org/eclipse/leshan/server/observation/ObservationListener.java

```diff
@@ -30,5 +30,13 @@
      * @param observation the observation for which new data are received
      * @param reponse the lwm2m response received
      */
-    void newValue(Observation observation, ObserveResponse response);
+    void onResponse(Observation observation, ObserveResponse response);
```

Comment:
That's right. I tried to have a similar approach than our current [send API](https://github.com/eclipse/leshan/blob/master/leshan-server-core/src/main/java/org/eclipse/leshan/server/LwM2mServer.java#L79).